tests/ml/tensorflow_test.py

```diff
@@ -0,0 +1,112 @@
+import logging
+
+import pytest
+pytest.importorskip("tensorflow")
+pytest.importorskip("tensorflow_io")
+
+import tensorflow as tf
+
+import vaex
+import vaex.ml
+
+
+
+logger = tf.get_logger()
+logger.setLevel(logging.ERROR)
+
+
+# Set up the data to be used in this test file
+df = vaex.ml.datasets.load_iris()
+df_train, df_test = df.ml.train_test_split(test_size=0.2, verbose=False)
+features = df.column_names[:4]
+target = 'class_'
+
+
+def test_to_dataset_tensorflow():
+    ds = df_test.ml.tensorflow.to_dataset(features=features, target=target, chunk_size=10)
+    list_ds = list(ds)
+    assert len(list_ds) == 3  # The number of "batches" in the iterable, as defined by the chunk_size arg
+
+    idx_min = 0
+    idx_max = 10
+    for batch in list_ds:
+        assert len(batch) == 2  # check that each batch has feature matrix and target matrix (hence tuple of len 2)
+        # Assert the target
```

Comment:
These comments don't add anything, they are only harmful, since they don't explain anything, and will lie in the future if we forgot to update it.

Reply:
Well, we disagree about the use of comments consistently so.. To me the code without the comments is a bit hard to follow is I naively would not know what to expect, so the comments are just reminders of that the lines are meant to accomplish. 

It is easy to remove, i have no problem with it.

Reply:
Removed most of the comments